src/test/java/roomescape/acceptance/ReservationAcceptanceTest.java

```diff
@@ -0,0 +1,86 @@
+package roomescape.acceptance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import roomescape.controller.dto.ReservationRequest;
+import roomescape.controller.dto.ReservationResponse;
+import roomescape.controller.dto.ReservationTimeResponse;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Sql(scripts = "/truncate.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+class ReservationAcceptanceTest {
+
+    @LocalServerPort
+    private int port;
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @BeforeEach
+    public void setUp() {
+        RestAssured.port = port;
+        insertDefaultData();
+    }
+
+    private void insertDefaultData() {
+        jdbcTemplate.update("INSERT INTO reservation_time (start_at) VALUES ?", "15:00");
+        jdbcTemplate.update("INSERT INTO reservation (name, date, time_id) VALUES (?, ?, ?)",
+                "브라운", "2023-08-05", 1L);
```

Comment:
```suggestion
        jdbcTemplate.update(
            "INSERT INTO reservation (name, date, time_id) VALUES (?, ?, ?)", 
            "브라운", "2023-08-05", 1L);
```
줄바꿈을 윗줄에도 해줘서 가독성을 높여볼 수 있을 것 같네요!